src/cascadia/Remoting/Monarch.cpp

```diff
@@ -1132,16 +1136,14 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
         }
         else
         {
+            // We couldn't find the peasant that started the drag. Well that

```

Comment:
Curious: this wouldn't occur when you try and drag the last tab in a window?

Reply:
Nah cause the originating terminal tab doesn't actually close until the drag is completed. Once the drag is accepted, the original window detaches from the old content and closes the tab, and at that point the new window already has a reference to the content it wants to open.